src/HumiditySensorDevice.ts

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env node
+
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { singleton } from "./util/Singleton";
+import { Time } from "./time/Time";
+import { TimeNode } from "./time/TimeNode";
+import { execSync } from "child_process";
+
+
+Time.get = singleton(() => new TimeNode());
+
+import { MatterDevice } from "./matter/MatterDevice";
+import { UdpInterface } from "./net/UdpInterface";
+import { SecureChannelProtocol } from "./matter/session/secure/SecureChannelProtocol";
+import { PaseServer } from "./matter/session/secure/PaseServer";
+import { Crypto } from "./crypto/Crypto";
+import { CaseServer } from "./matter/session/secure/CaseServer";
+import { ClusterServer, InteractionServer } from "./matter/interaction/InteractionServer";
+import { BasicInformationCluster } from "./matter/cluster/BasicInformationCluster";
+import { GeneralCommissioningCluster, RegulatoryLocationType } from "./matter/cluster/GeneralCommissioningCluster";
+import { OperationalCredentialsCluster } from "./matter/cluster/OperationalCredentialsCluster";
+import { DEVICE } from "./matter/common/DeviceTypes";
+import { MdnsBroadcaster } from "./matter/mdns/MdnsBroadcaster";
+import { Network } from "./net/Network";
+import { NetworkNode } from "./net/node/NetworkNode";
+import { commandExecutor } from "./util/CommandLine";
+import { getParameter } from "./util/CommandLine";
+import { WaterContentMeasurementCluster } from "./matter/cluster/WaterContentMeasurementCluster";
+import { GeneralCommissioningClusterHandler } from "./matter/cluster/server/GeneralCommissioningServer";
+import { OperationalCredentialsClusterHandler } from "./matter/cluster/server/OperationalCredentialsServer";
+import { MdnsScanner } from "./matter/mdns/MdnsScanner";
+import packageJson from "../package.json";
+import { Logger } from "./log/Logger";
+import { VendorId } from "./matter/common/VendorId";
+import { ByteArray } from "@project-chip/matter.js";
+import { CommissionningFlowType, DiscoveryCapabilitiesSchema, ManualPairingCodeCodec, QrPairingCodeCodec } from "./codec/PairingCode.js";
+import { QrCode } from "./codec/QrCode.js";
+import { NetworkCommissioningCluster, NetworkCommissioningStatus } from "./matter/cluster/NetworkCommissioningCluster";
+import { AdminCommissioningCluster, CommissioningWindowStatus } from "./matter/cluster/AdminCommissioningCluster";
+import { AdminCommissioningHandler } from "./matter/cluster/server/AdminCommissioningServer";
+import { NetworkCommissioningHandler } from "./matter/cluster/server/NetworkCommissioningServer";
+import { FabricIndex } from "./matter/common/FabricIndex";
+
+// From Chip-Test-DAC-FFF1-8000-0007-Key.der
```

Comment:
Would be good to extract the 4 constants below in its own file and reuse it.

Yes, those certificates should autogenerated insteda of using CHIP example certificates but the code to autogenerate them is not written yet (should not be **too** complicated since this is standard crypto).

Reply:
Moved certs into own file. Will update code to include

Reply:
Maybe call the File DevCertificates.ts or such?